.github/workflows/comment-pr.yml

```diff
@@ -0,0 +1,56 @@
+# Description: This workflow is triggered when the `receive-pr` workflow completes to post suggestions on the PR.
+# Since this pull request has write permissions on the target repo, we should **NOT** execute any untrusted code.
+# https://securitylab.github.com/research/github-actions-preventing-pwn-requests/
+---
+name: comment-pr
+
+on:
+  workflow_run:
+    workflows: ["receive-pr"]
+    types:
+      - completed
+
+jobs:
+  post-suggestions:
+    # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-a-workflow-based-on-the-conclusion-of-another-workflow
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    runs-on: ubuntu-latest
+    env:
+      # https://docs.github.com/en/actions/reference/authentication-in-a-workflow#permissions-for-the-github_token
+      ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{github.event.workflow_run.head_branch}}
+          repository: ${{github.event.workflow_run.head_repository.full_name}}
+
+      # Download the patch
+      - uses: actions/download-artifact@v4
+        with:
+          name: patch
+          github-token: ${{ secrets.GH_PAT_ACTIONS_READ }}
```

Comment:
This secret does not yet exist, and I of course don't have access to create it for this organization. It should be a GitHub personal access token with `actions:read` permissions on the target repo. This is used to download the patch and the PR number from the triggering workflow. Fairly innocuous, but unfortunately a manual step once.

Reply:
Looks like this also works with GITHUB_TOKEN, so this comment can be discarded after https://github.com/langchain4j/langchain4j/pull/673/commits/225b8603a820e4e6a7152df5695635c03f30c4f5